python/dgl/dataloading/dataloader.py

```diff
@@ -351,26 +378,34 @@ def collate(self, items):
 
             If the original graph has multiple node types, return a dictionary of
             node type names and node ID tensors.  Otherwise, return a single tensor.
+        items : Tensor or dict[ntype, Tensor], optional
```

Comment:
The name items is misleading. This should be the indices.

Reply:
Further, in Parameters, the items can also be indices.

Reply:
I'm not sure if the argument name should be indices.  It can also represent the actual node IDs if `return_indices` is False.

Reply:
It is returned only when return_indices is True

Reply:
It's right below.